certified-connectors/3E Events/readme.md

```diff
@@ -0,0 +1,40 @@
+[[_TOC_]]
```

Comment:
@isalzh Can you please let us know the purpose of the above line?

Reply:
It's table of contents markdown. https://docs.microsoft.com/en-us/azure/devops/project/wiki/wiki-markdown-guidance?view=azure-devops
Looks like it's not supported by GitHub and we missed that fact. Fill fix, thank you.

Reply:
@isalzh Please follow the template we have provided for `"readme.md"` file and update this file by removing the TOC line.

https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/certified-connectors/readme.md

Reply:
removed